src/omv/boards/OPENMV_RT1060/omv_boardconfig.h

```diff
@@ -18,6 +18,11 @@
 #define OMV_BOARD_UID_SIZE              3            // Unique ID size in words.
 #define OMV_BOARD_UID_OFFSET            12           // Bytes offset for multi-word UIDs.
 
+// RAW Preview configuration.
+#define OMV_RAW_PREVIEW_ENABLE          (1)
+#define OMV_RAW_PREVIEW_WIDTH           (800)
+#define OMV_RAW_PREVIEW_HEIGHT          (600)
```

Comment:
```suggestion
#define OMV_RAW_PREVIEW_WIDTH           (800)
#define OMV_RAW_PREVIEW_HEIGHT          (600)
```
That's not really the purpose of this feature. Even if bandwidth allow it, you'd be spending a lot of time moving huge buffers to USB. The main purpose of this feature is to free CPU time. If you can't make this smaller you don't have to use it.

Reply:
If it's jpeg compressed at a larger resolution, it takes more CPU time than just memcpying. Same for scaling. You kinda want this size to be as large as possible to avoid doing anything other than a memcpy as this is the fast CPU operation.

Yes, this does make the image transfer size larger, but, you're getting a better trade over how long it would take the CPU to process the frame via scaling or JPEG.

Reply:
None of the boards use DMA to copy buffers to USB, not even the ST stack by default, but even if they did it will still consume a lot of the chip's bus bandwidth. So the CPU has to copy ~234 x 4KBs buffers per frame which can't be free. The trade-off is JPEG compression time for smaller preview image, not JPEG compression time for CPU time and bus bandwidth.

Reply:
The conditonals for this are somewhat complex on what is best. Will re-enable after PXP is working for the RT1062 which does hardware scaling.

An example, the camera FPS for turning this feature off or on is the same for SVGA and below. However, the IDE FPS changes and increases with this feature enabled.

Once you go above the RAW res limit and start scaling you get behavior that affects the camera FPS. Handling 1280x720 without this feature limits the camera FPS to 9.5, and with it, the system FPS goes up to 16. IDE FPS is then about half or each. So, in this case scaling is cheaper than JPEG compression which is so high it affects system performance while connected to the IDE.

Given that, the question then becomes, if I reduced the preview width/height, would this increase the camera FPS (not IDE FPS). Maxing out the scaling performance is required to find the answer to this as it is a cheaper operation than JPEG.

Reply:
Dropped